src/components/Comments/WriteComment.js

```diff
@@ -242,7 +241,7 @@ export const styles = StyleSheet.create({
   connectorBottom: (inReplyCommentView, showProfileConnector) => ({
     height: showProfileConnector ? 20 : 0,
     width: 1,
-    backgroundColor: colors.gray1,
+    backgroundColor: COLORS.gray9,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `connectorTop` and `connectorBottom` styles has been updated. Ensure that this new color is consistent with your design system.
```diff
- backgroundColor: colors.gray1,
+ backgroundColor: COLORS.gray9,
...
- backgroundColor: colors.gray1,
+ backgroundColor: COLORS.gray9,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->